source/code-snippets/usage-examples/updateOne.js

```diff
@@ -1,34 +1,45 @@
+/* Update a document */
```

Comment:
I think we agreed to just do // for single-line comments. Otherwise this isn't something that would hold up "approval"